fastify.d.ts

```diff
@@ -159,6 +158,7 @@ export interface ValidationResult {
 }
 
 /* Export all additional types */
+export { Chain as LightMyRequestChain, InjectOptions, Response as LightMyRequestResponse, CallbackFunc as LightMyRequestCallback } from 'light-my-request'
```

Comment:
Export same as import in https://github.com/fastify/fastify/blob/main/types/instance.d.ts#L1

Reply:
I would suggest using the syntax below as we only re-export the type.
```
export type { .. } from 'light-my-request'
```

Reply:
Good point